pkg/terraform/executor.go

```diff
@@ -531,7 +532,7 @@ func (ex *Executor) checkVersion() error {
 
 	v, err := version.NewVersion(vStr)
 	if err != nil {
-		return fmt.Errorf("checking Terraform version: %w", err)
+		return fmt.Errorf("initializing version object from %q: %w", vStr, err)
```

Comment:
I don't know as `NewVersion` parses and returns a new version, maybe we can do. I am fine with initializing too 

```suggestion
		return fmt.Errorf("parsing version object from %q: %w", vStr, err)
```

Reply:
I've change it to yet something else. Please have a look.